repository/src/main/java/org/apache/atlas/repository/graph/GraphBackedSearchIndexer.java

```diff
@@ -393,7 +398,7 @@ private void initialize(AtlasGraph graph) throws RepositoryException, IndexExcep
             createCommonVertexIndex(management, TASK_GUID, UniqueKind.GLOBAL_UNIQUE, String.class, SINGLE, true, false);
             createCommonVertexIndex(management, TASK_TYPE_PROPERTY_KEY, UniqueKind.NONE, String.class, SINGLE, true, false);
             createCommonVertexIndex(management, TASK_CREATED_TIME, UniqueKind.NONE, Long.class, SINGLE, true, false);
-            createCommonVertexIndex(management, TASK_STATUS, UniqueKind.NONE, String.class, SINGLE, true, false);
+            createCommonVertexIndex(management, TASK_STATUS, UniqueKind.NONE, String.class, SINGLE, true, false, true, new HashMap<>(), KEYWORD_FIELD);
```

Comment:
Do not change `isStringField` for `TASK_STATUS`, keep it false to be backward compatible with existing tenants, it should be `default:text` & `keyword` as field

```
"__task_status": {
    "type": "text",
    "fields": {
        "keyword": {
            "type": "keyword"
        }
    },
    "copy_to": [
        "all"
    ]
}